src/contrib/cluster/Akka.Cluster.Tools.Tests/PublishSubscribe/DistributedPubSubMediatorSpec.cs

```diff
@@ -58,47 +59,27 @@ public void DistributedPubSubMediator_can_be_activated_in_child_actor()
         }
 
         [Fact]
-        public void DistributedPubSubMediator_should_get_dead_letter()
+        public async Task DistributedPubSubMediator_should_send_messages_to_dead_letter()
         {
-            EventFilter.Exception<NullReferenceException>().Expect(0, () =>
+            await EventFilter.DeadLetter<object>().ExpectAsync(10, TimeSpan.FromMinutes(3), () =>
```

Comment:
The default `removed-time-to-live` is 2 minutes - that is when pruning takes effect if there are no subscribers to a Topic. Hence, the reason for the `TimeSpan.FromMinutes(3)`